setup.cfg

```diff
@@ -1,3 +1,64 @@
+[metadata]
+name = awkward
+url = https://github.com/scikit-hep/awkward-1.0
+project_urls =
+  Documentation = https://awkward-array.org
+  Source Code = https://github.com/scikit-hep/awkward-1.0
+  Bug Tracker = https://github.com/scikit-hep/awkward-1.0/issues
+  Discussions = https://github.com/scikit-hep/awkward-1.0/discussions
+  Chat = https://gitter.im/Scikit-HEP/awkward-array
+  Releases = https://github.com/scikit-hep/awkward-1.0/releases
```

Comment:
Other packages I was looking at often fill these out, it makes nice links on the sidebar.

Reply:
Okay!